recipes/plant_tribes_assembly_post_processor/1.0.4/meta.yaml

```diff
@@ -0,0 +1,33 @@
+package:
+  name: plant_tribes_assembly_post_processor
+  version: "1.0.4"
+
+source:
+  url: https://github.com/dePamphilis/PlantTribes/archive/v1.0.4.tar.gz
+  md5: b86b33dbf0766c8b2c2de75641dee65b
+
+build:
+  number: 0
+  # Requires genometools which is not supported on osx
+  skip: True # [osx]
+
+requirements:
+  run:
+    - cap3
+    - genometools-genometools >=1,<2
+    - hmmer >=3
+    - mafft >=7,<8
+    - perl >=5.22
+    - perl-estscan2 2.1 pl5.22.0_1
```

Comment:
@gregvonkuster is this really needed with the strict pinning? We try to get rid of those old perl version.